src/components/TodoForm.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { Modal, Center } from "@mantine/core";
+
+function TodoForm(props) {
+  const [input, setInput] = useState("");
+  const [update, setUpdate] = useState(props.edit ? props.edit.value : "");
+
+  const handleChange = (e) => {
+    if (props.editMode) setUpdate(e.target.value)
+    else setInput(e.target.value);
```

Comment:
if you use return statements, the else is redundant here.
```suggestion
    if (props.editMode) return setUpdate(e.target.value)
    setInput(e.target.value);
```